elasticsearch/Dockerfile

```diff
@@ -31,7 +31,8 @@ RUN rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch && \
     yum clean all
 # initial boring elasticsearch.yml is just to enable plugin installation.
 # put the real config in place in a deployment with a ConfigMap.
-ADD elasticsearch.yml logging.yml /usr/share/elasticsearch/config/
+ADD elasticsearch.yml /usr/share/elasticsearch/config/
+ADD index_templates /usr/share/elasticsearch/config/index_templates
```

Comment:
If you add `shopt -o failglob` at the beginning it will affect everything.  If you are absolutely sure that's safe, that's fine.  If not, then add `shopt -o failglob` before the loop, and add `shopt -o nofailglob` after the loop.